### ISSUE TITLE
Example not working

### ISSUE THREAD

**Issue by miguelmota** [pedrouid/starkware-crypto#2]
Example code from README

```js
const starkwareCrypto = require('starkware-crypto')

const privateKey =
  '0x659d82c1cc4c3e6fead938999322116a3dc7854b415b822dbea42630ecd90b5e'

const keyPair = starkwareCrypto.getKeyPair(privateKey)

const publicKey = starkwareCrypto.getPublic(privateKey)

const starkKey = starkwareCrypto.getStarkKey(publicKey)

console.log(starkKey)
```

```bash
$ node index.js

/home/mota/Sandbox/test/node_modules/starkware-crypto/dist/cjs/index.js:122
    return keyPair.getPublic(true, 'hex');
                   ^

TypeError: keyPair.getPublic is not a function
    at Object.getPublic (/home/mota/Sandbox/test/node_modules/starkware-crypto/dist/cjs/index.js:122:20)
    at Object.<anonymous> (/home/mota/Sandbox/test/starkex-test/index.js:8:35)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:877:12)
    at internal/main/run_main_module.js:21:11
```

**Comment by pedrouid** [comment]
Ooppss... Sorry I've updated the API and forgot to update the example

**Comment by pedrouid** [comment]
Fixed README example, added it to the tests and published it as v1.6.0

**Comment by miguelmota** [comment]
@pedrouid thanks!
